src/test/java/pdfParsers/PdfToJsonConverterTest.java

```diff
@@ -0,0 +1,121 @@
+package pdfParsers;
+
+import loaders.LocalPdfReader;
+
+import java.io.InputStream;
+import java.io.IOException;
+
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.JsonParser;
+import org.codehaus.jackson.map.MappingJsonFactory;
+import org.codehaus.jackson.node.ArrayNode;
+import org.junit.Test;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class PdfToJsonConverterTest {
+    
+    @Test
+    public void checkboxPdfHasExpectedFields() throws Exception {
+        assertThat(getFieldsFromPdf("checkbox.pdf"),
+            contains(
+                allOf(
+                    hasJsonProperty("name", equalTo("Check Box2")),
+                    hasJsonProperty("value", equalTo("")),
+                    hasJsonProperty("type", equalTo("checkbox")),
+                    hasJsonProperty("tabIndex", equalTo(0)),
+                    hasJsonProperty("required", equalTo(false)),
+                    hasJsonProperty("altText", equalTo(""))
```

Comment:
I wasn't sure whether all of these fields are equally important for our use cases (for instance, does `required` or `altText` matter when we're filling out a form?) but I figured we may as well verify all of them, as long as we're testing the overall behavior of `PdfToJsonConverter`. But I left out the properties within `position`, since those seemed like formatting details that might not be so reliable.